model/SupplyChain/SupplyChain.md

```diff
@@ -10,6 +10,10 @@ The SupplyChain profile documents events and processes associated with the lifec
 
 The SupplyChain profile documents events and processes associated with the lifecyle of a product, including its creation, transportation, usage, and decommissioning.
 
+In this context, the term **“product”** is used in a broad sense to encompass not only conventional finished goods, but also raw materials classified as **commodities** and various forms of **assets** that hold economic value, including software. 
```

Comment:
Overall - the description reads clear to me an it looks like it is distinct from the "Package" concept in that it is from the consumer perspective and includes "goods" which may not have been produced for distribution (e.g. commodities).

One suggestions - this description introduces another term which is not defined in the spec "goods".  I would prefer we, at some point, relate the definition back to a term defined in the spec.  Are "goods" always `Artifact`s?  If so, we could replace "goods" with "Artifacts" to resolve my concern.  If "goods" are not always artifacts, we could add something that describes "goods" as a type of SPDX element.